core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -39,18 +39,17 @@ package object config {
   private[spark] val SPARK_RESOURCES_COORDINATE =
     ConfigBuilder("spark.resources.coordinate.enable")
       .doc("Whether to coordinate resources automatically among workers/drivers(client only) " +
-        "in Standalone. If not, user should be responsible for assigning different resources " +
-        "for workers/drivers while using resource discovery script.")
+        "in Standalone. If false, the user is responsible for configuring different resources " +
+        "for workers/drivers that run on the same host.")
       .booleanConf
-      .createWithDefault(true)
+      .createWithDefault(false)
```

Comment:
did you mean to turn this to false?  If so we need to update the configuration.md to match.  I'm ok either way

Reply:
Oops. I misunderstand your comment. I'd prefer it to be `true`. And tests' failures are probably affected by this change.